docs/design-docs/benchmarks.md

```diff
@@ -244,6 +249,12 @@ result = {
             'throughput-inference-float32': [avg_throughput1, ..., avg_throughputN],
             'throughput-inference-float16': [avg_throughput1, ..., avg_throughputN],
     },
```

Comment:
Do we need to change result as well?

Reply:
Here the result is from one executor, it don't need to change. `avg `here means the average of every step.

Reply:
I think the result should be min already. We don't have average anymore.

Reply:
Got it. It's for executor output.